build-system-tests/templates/components/react/cra/App.js

```diff
@@ -45,7 +52,9 @@ export default function Home() {
         maxFileCount={1}
         isResumable
       />
-      <MapView />
+      <MapView>
+        <Marker latitude={latitude} longitude={longitude} />
```

Comment:
Should we include a `LocationSearch` component in these tests? any reason not to?

Reply:
+1

Reply:
I don't know if we need to include `LocationSearch` for build test, but I added it just in case. Thanks for checking!